packages/fast-components/src/checkbox/checkbox.template.ts

```diff
@@ -0,0 +1,54 @@
+import { html, when } from "@microsoft/fast-element";
+import { Checkbox } from "./checkbox";
+import { bool } from "../utilities";
+
+export const CheckboxTemplate = html<Checkbox>`
+    <div
+        part="checkbox"
+        role="checkbox"
+        class="checkbox"
+        $aria-checked="${x => bool(x.checked)}"
+        $aria-required="${x => bool(x.required)}"
+        $aria-disabled="${x => bool(x.disabled)}"

```

Comment:
Should these use the latest work Rob built to pass to the host?

Reply:
I don't think he has added the functionality for that, but I've hoisted these up to the host with imperative APIs